tornadovmInstaller.sh

```diff
@@ -277,19 +346,47 @@ function installForWindowsJDK11() {
     setupVariables $dirname
 }
 
+function installForWindowsJDK16() {
+    checkPrerequisites
+    dirname="TornadoVM-Windows-JDK16"
+    mkdir -p $dirname
+    cd $dirname
+    downloadWindowsJDK16
+    downloadCMake
+    setupTornadoVM jdk-11-plus
+    setupVariables $dirname
+}
+
 function printHelp() {
     echo "TornadoVM installer for Linux and OSx"
     echo "Usage:"
     echo "       --jdk8           : Install TornadoVM with OpenJDK 8  (Default)"
-    echo "       --graal-jdk-8    : Install TornadoVM with GraalVM and JDK 8 (GraalVM 20.2.0)"
-    echo "       --graal-jdk-11   : Install TornadoVM with GraalVM and JDK 11 (GraalVM 20.2.0)"
+    echo "       --graal-jdk-8    : Install TornadoVM with GraalVM and JDK 8 (GraalVM 21.1.0)"
+    echo "       --graal-jdk-11   : Install TornadoVM with GraalVM and JDK 11 (GraalVM 21.1.0)"
+    echo "       --graal-jdk-16   : Install TornadoVM with GraalVM and JDK 16 (GraalVM 21.1.0)"
     echo "       --corretto-11    : Install TornadoVM with Corretto JDK 11"
-    echo "       --mandrel-11     : Install TornadoVM with Mandrel 20.2.0 (JDK 11)"
+    echo "       --corretto-16    : Install TornadoVM with Corretto JDK 16"
+    echo "       --mandrel-11     : Install TornadoVM with Mandrel 21.1.0 (JDK 11)"
     echo "       --windows-jdk-11 : Install TornadoVM with Windows JDK 11"
+    echo "       --windows-jdk-16 : Install TornadoVM with Windows JDK 16"
+    echo "       --opencl         : Install TornadoVM and build the OpenCL backend"
+    echo "       --ptx            : Install TornadoVM and build the PTX backend"
     echo "       --help           : Print this help"
     exit 0
 }
 
+function setBackend() {
+for i in ${args[@]}
+do
+  flag=$i
+  if [[ "$flag" == '--opencl' ]]; then
+    opencl=true
+  elif [[ "$flag" == '--ptx' ]]; then
+    ptx=true
+  fi
```

Comment:
I think we do not need the flag to be true, because if a user wants to build TornadoVM with `make`, the Makefile sets the default backend to be `opencl`. I've tested it. 

However, as logic in the scripts we can have `opencl` as the default. What do you think?

Reply:
Yes, can you add opencl as default here?